server/backend/migration/20231003000000.go

```diff
@@ -0,0 +1,223 @@
+	file, err := staticData.ReadFile("static_data/dishNameTags.json")
+	if err != nil {
+		log.WithError(err).Error("Error including json.")
+	}
+
+	var tagsNames multiLanguageNameTags
+	errjson := json.Unmarshal(file, &tagsNames)
+	if errjson != nil {
+		log.WithError(errjson).Error("Error parsing nameTagList to json.")
+	}
+	for _, v := range tagsNames.MultiLanguageNameTags {
+		parent := DishNameTagOption{
+			DE: v.TagNameGerman,
+			EN: v.TagNameEnglish,
+		}
+
+		if err := db.Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).Create(&parent).Error; err != nil {
+			fields := log.Fields{"en": v.TagNameEnglish, "de": v.TagNameGerman}
+			log.WithError(err).WithFields(fields).Error("Error while creating tag")
+		}
+
+		addCanBeIncluded(parent.DishNameTagOption, db, v)
+		addNotIncluded(parent.DishNameTagOption, db, v)
+	}
+}
+
+func addNotIncluded(parentId int64, db *gorm.DB, v nameTag) {
+	for _, expression := range v.NotIncluded {
+		fields := log.Fields{"expression": expression, "parentId": parentId}
+		err := db.
+			Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).
+			Create(&DishNameTagOptionExcluded{
+				Expression: expression,
+				NameTagID:  parentId}).Error
+		if err != nil {
+			log.WithError(err).WithFields(fields).Error("Unable to create new DishNameTagOptionExcluded")
+		}
+	}
+}
+
+func addCanBeIncluded(parentId int64, db *gorm.DB, v nameTag) {
+	for _, expression := range v.CanBeIncluded {
+		fields := log.Fields{"expression": expression, "parentId": parentId}
+
+		err := db.
+			Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).
+			Create(&DishNameTagOptionIncluded{
+				Expression: expression,
+				NameTagID:  parentId,
+			}).Error
+		if err != nil {
+			log.WithError(err).WithFields(fields).Error("Unable to create new can be excluded tag")
+		}
+	}
+}
+
+/*
+Reads the json file at the given path and checks whether the values have already been inserted into the corresponding table.
+If an entry with the same German and English name exists, the entry won't be added.
+The TagType is used to identify the corresponding model
+*/
+func setTagTable(path string, db *gorm.DB, tagType backend.ModelType) {
+	tagsDish := generateRatingTagListFromFile(path)
+
+	var insertModel *gorm.DB
+	if tagType == backend.DISH {
+		insertModel = db.Model(&DishRatingTagOption{})
+	} else {
+		insertModel = db.Model(&CafeteriaRatingTagOption{})
+	}
+
+	for _, v := range tagsDish.MultiLanguageTags {
+		fields := log.Fields{"de": v.TagNameGerman, "en": v.TagNameEnglish}
+		var err error
+		if tagType == backend.CAFETERIA {
+			element := CafeteriaRatingTagOption{
+				DE: v.TagNameGerman,
+				EN: v.TagNameEnglish,
+			}
+			err = insertModel.Create(&element).Error
+		} else {
+			element := DishRatingTagOption{
+				DE: v.TagNameGerman,
+				EN: v.TagNameEnglish,
+			}
+			err = insertModel.Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).Create(&element).Error
+		}
```

Comment:
this `.Model(..)` does not seem nessesary accordin to https://gorm.io/docs/create.html

is it intentional that the if condition is different?

```suggestion
	for _, v := range tagsDish.MultiLanguageTags {
		fields := log.Fields{"de": v.TagNameGerman, "en": v.TagNameEnglish}
		var err error
		if tagType == backend.CAFETERIA {
			element := CafeteriaRatingTagOption{
				DE: v.TagNameGerman,
				EN: v.TagNameEnglish,
			}
			err = db.Create(&element).Error
		} else {
			element := DishRatingTagOption{
				DE: v.TagNameGerman,
				EN: v.TagNameEnglish,
			}
			err = db.Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).Create(&element).Error
		}
```